helper.py

```diff
@@ -0,0 +1,76 @@
+#from bety_brapi import database
+
+import database
```

Comment:
not sure we need to use this file, again don't just follow the BRAPI specs, look at some other ones as well (good example is the petstore).

Reply:
removed